### PR TITLE
Fixes styles for reaction in a table and footer-related

### PR DESCRIPTION
**Pull request by mike-lvov**
Fixes previously broken footer styles. Adds a few style tweaks to the reactions

### CODE REVIEW

File: src/components/organisms/Room/Participant.tsx [hunk 1]
```diff
@@ -135,6 +135,7 @@ const Participant: React.FC<React.PropsWithChildren<ParticipantProps>> = ({
           <UserProfilePicture
             user={{ ...profileData, id: participant.identity }}
             setSelectedUserProfile={() => setShowProfile(true)}
+            reactionPosition="right"
```
Comment (0xdevalias) [inline src/components/organisms/Room/Participant.tsx:138]: This looked fine when the participant was on both the left of the screen and the right of the screen yeah?

**Review by 0xdevalias — APPROVED** [review]
Code changes look good, and screenshots on slack looked good. LGTM!

File: src/components/organisms/Room/Participant.tsx (hunk 1 above)
Comment (mike-lvov) [reply src/components/organisms/Room/Participant.tsx:138]: For small to medium-sized messages yes

Comment (0xdevalias) [reply src/components/organisms/Room/Participant.tsx:138]: Awesome

**Pull request merged**